src/modules/dashboard/components/Members/Members.tsx

```diff
@@ -48,6 +49,19 @@ const MSG = defineMessages({
     id: 'dashboard.Members.failedToFetch',
     defaultMessage: "Could not fetch the colony's members",
   },
+  contributorsTitle: {
```

Comment:
I would prefer if you moved the title and description into the `MembersSection` component and pass less props. `Members` component is already huge and the goal is to move as much as possible from it.